apiary.apib

```diff
@@ -1682,6 +1682,364 @@ asynchronously once the job is created.
             }
 
 
+# Group Factory API
+
+The Factory API allows you to interact with orders that have been placed and move them through the various
+stages of production. The typical workflow is as follows:
+
+<a href="https://github.com/unmadeworks/api-docs/assets/13105806/c638182b-ab00-4141-96de-e4fe77de2a1f">
+  <img src="https://github.com/unmadeworks/api-docs/assets/13105806/c638182b-ab00-4141-96de-e4fe77de2a1f" alt="Image showing the typical user journey through the e-commerce site" width="100%">
+</a>
+
+
+#### Factory Slugs
+
+All Factory API endpoints begin with `/v2/factories/<factory_slug>` where `factory_slug` is the unique identifier
+for each of your factory locations. These slugs will be provided to you by Unmade. Each API token provided to you
+can have access to **one or more individual factory locations**, depending on your specific requirements.
+
+#### Order States
+
+During production, orders progress through the following states:
+
+ - `new`: The order has not yet begun production
+ - `in_production`: The order has been accepted and is now in production
+ - `ready-to-ship`: The order has been produced and is awaiting shipping
+ - `shipped`: The order has left the factory
+ - `cancelled`: The order has been cancelled
```

Comment:
I'm using the Factory OMS UI states here to match the UI. I'm expecting to pass the filter querystring right down to Factory OMS and let Factory work it out (rather than filtering the Embed orders).

Reply:
hmmm, it is dubious that the states don't match. But then again, you could argue that the factory states are factory agnostic. The only one that doesn't match is submitted?

Reply:
Embed also has `draft`, `new`, `hold` and  `submitting` which all mean "this order isn't on Factory OMS yet". And then `submitted` which roughly corresponds to Factory OMS `new`. I don't really know what to do about this other than try and explain that the states between these two endpoints are different.

But hopefully it's not a big problem. From a Factory API user's point of view, they'd n

Reply:
The way that I see it is that `submitted`  on embed is the only odd one because at the order exists in another system with a different state. Now that you mention it, `new` on embed should have a name change.

Reply:
`new` and `submitting` basically never get seen. And a customer would never filter looking for those states specifically.

So if anything, we could display `new` when the Embed state is `submitted`. Then it would basically match

Reply:
that sounds like a good idea!